flink-core/src/main/java/org/apache/flink/configuration/RestartStrategyOptions.java

```diff
@@ -222,6 +223,19 @@ public class RestartStrategyOptions {
                                             code(RESTART_STRATEGY.key()), code("exponential-delay"))
                                     .build());
 
+    @Documentation.OverrideDefault("infinite")
+    public static final ConfigOption<Integer> RESTART_STRATEGY_EXPONENTIAL_DELAY_ATTEMPTS =
+            ConfigOptions.key("restart-strategy.exponential-delay.attempts-before-reset-backoff")
+                    .intType()
+                    .defaultValue(Integer.MAX_VALUE)
+                    .withDescription(
+                            Description.builder()
+                                    .text(
+                                            "The number of times that Flink retries the execution before the job is declared as failed "
+                                                    + "before resetting the backoff to its initial value if %s has been set to %s.",
```

Comment:
Maybe separate the long statement to make it easier for reading? e.g.
```
The number of times that Flink retries the execution before failing the job if %s has been set to %s. The number will be reset once the backoff is reset to its initial value.
```

Reply:
Thanks @zhuzhurk for the great suggestion! It's easier to understand~

I updated the original commit directly due to this change is easy enough, and the change can be viewed by this link[1].

[1] https://github.com/apache/flink/compare/f91f422f3350b622715d140a1255746aafb6a1e7..08efd3f4e062490ac2431408bcaf3d079cf1e6bb